analyzers/src/SonarAnalyzer.CSharp/Helpers/CountComparisonSyntaxHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace SonarAnalyzer.Helpers
+{
+    public static class CountComparisonSyntaxHelper
+    {
+        private static readonly IExpressionNumericConverter Converter = new CSharpExpressionNumericConverter();
+
+        public static CountComparisonSyntax CountComparison(this BinaryExpressionSyntax binary, SemanticModel semanticModel)
+        {
+            int? left = default;
+            int? right = default;
+            if (Converter.TryGetConstantIntValue(binary.Left, semanticModel, out var l_out))
+            {
+                left = l_out;
+            }
+            if (Converter.TryGetConstantIntValue(binary.Right, semanticModel, out var r_out))
+            {
+                right = r_out;
+            };
+            return new CountComparisonSyntax(left, binary.Operator(), right);
+        }
+        private static CountComparisonSyntax.Kind Operator(this CSharpSyntaxNode syntax) =>
```

Comment:
_Nitpick:_ Missing EOL
```suggestion
        }
        
        private static CountComparisonSyntax.Kind Operator(this CSharpSyntaxNode syntax) =>
```